SPECS/sm.spec

```diff
@@ -431,6 +431,11 @@ then
 fi
 
 %changelog
+* Fri Jul  4 2025 Yann Dirson <yann.dirson@vates.fr> - 3.2.12-3.3
```

Comment:
```suggestion
* Fri Jul 04 2025 Yann Dirson <yann.dirson@vates.fr> - 3.2.12-3.3
```

Reply:
That's called rpmlint, but it needs to be fine-tuned to be actually useful.